### ISSUE TITLE
"Illegal measurement mode 1" for count_pulses() and compute_tofs()

### ISSUE THREAD

**Issue by sebrit** [HowardALandman/QTD#9]
compute_tofs() and count_pulses() fails with "Illegal measurement mode 1" if during configuration meas_mode=1 is set.
Both functions compare self.meas_mode (=1) with the register value for the selection of the measurement mode self._CF1_MM1 (=0).

Tested on TDC7200, but that should be irrelevant.

**Comment by HowardALandman** [comment]
Yeah, measurement mode 1 is not completely/correctly implemented, so I disabled it until I could fix that. Feel free to re-enable it for experimentation, but some aspects may not work or may give incorrect answers. My own research only requires MM2, so it’s gotten more attention.

Unfortunately (?) I am leaving on 8 weeks vacation today, and will not be able to work on this until mid-October. First stop is Iceland, where I hope to see the active volcano in Geldingsdalur tomorrow.

I should be able to clean up quite a few issues by January 10, when I have a major proposal due that uses the code.

    Howard A. Landman

**Comment by sebrit** [comment]
Alright, I'll mess around with MM1 a bit. This issue is comparatively easy to work around, e.g by modifying the check for the mode, so I have gotten reasonable results for the kind of measurements I want to do (time correlated single photon counting, so the measurement range is <100 ns).

Have a great vacation!

~Seb
